### ISSUE TITLE
Freeze when template contains double curly braces

### ISSUE THREAD

**Issue by meriadec** [mjmlio/mjml#487]
As raised here: https://github.com/mjmlio/mjml-app/issues/68, having `{{` in template results in a freeze.

e.g giving this code :
(tested with v3.1.1 and v3.2.0-beta.3)

```javascript
const mjml = require('mjml')

mjml.mjml2html(`
<mjml>
<mj-body>
  <mj-container>
    <mj-section>
      <mj-column>
        <mj-text>
          Hello world {{
        </mj-text>
      </mj-column>
    </mj-section>
  </mj-container>
</mj-body>
</mjml>
`)

console.log('after') // it never happens
```

No problem with a single curly brace.
Is there any workaround?

**Comment by lohek** [comment]
It's issue with cheerio (postRender and xmlParser)

The only workaround is to use numeral entities like
```xml
<mjml>
  <mj-body>
    <mj-container>
      <mj-section>
        <mj-column>
          <mj-text>
            Hello world &#123;&#123;
          </mj-text>
        </mj-column>
      </mj-section>
    </mj-container>
  </mj-body>
</mjml>
```

**Comment by meriadec** [comment]
After a bit digging, the problem is caused by a [juice](https://github.com/Automattic/juice) feature that try to replace `{{` and `}}` (interpreted as handlebars delimiters). See https://github.com/Automattic/juice/blob/master/lib/cheerio.js#L68 . The RegExp ends in an infinite loop..

IMHO, best solution would be to pass an option to prevent this behaviour in Juice. I'll see how to do that. Keep updated.

**Comment by adambullmer** [comment]
I had a same problem trying to fit templating syntax into my emails for translations and other dynamic, transactional content. The slight workaround I had was to close the double curly brackets _first_  and then open them
```
1. some text here
2. some text here }}
3. {{ some text here }}
```

Definitely nit ideal. but was an easy enough of a workaround for me.

**Comment by ngarnier** [comment]
Hi, thanks for reporting. Should be fixed in v3.3.0 (now in beta)

**Comment by iRyusa** [comment]
It should be fixed in 3.3.1 too !

**Comment by ngarnier** [comment]
Hello, 

Thanks for reporting @meriadec, it's now fixed (`npm install mjml` to update)!
